hathor/p2p/messages.py

```diff
@@ -90,10 +90,25 @@ class ProtocolMessages(Enum):
     GET_NEXT = 'GET-NEXT'
     NEXT = 'NEXT'
 
-    GET_BLOCKS = 'GET-BLOCKS'  # Request a list of hashes for blocks. Payload is the current latest block.
```

Comment:
The removed messages are not used in sync-v1?

Reply:
They aren't.